logic/peers.js

```diff
@@ -210,9 +227,10 @@ Peers.prototype.remove = function(peer) {
 };
 
 /**
- * Returns private list of peers
+ * Returns private list of peers.
+ *
  * @param {boolean} [normalize] - If true transform list to object
```

Comment:
As above.

Reply:
It indicates that the parameter is optional